discovery-provider/src/models.py

```diff
@@ -153,6 +152,8 @@ class User(Base):
     creator_node_endpoint = Column(String)
     updated_at = Column(DateTime, nullable=False)
     created_at = Column(DateTime, nullable=False)
+    primary = Column(Integer, nullable=True)
+    secondaries = Column(postgresql.ARRAY(Integer), nullable=True)
```

Comment:
just a thought -- do you think we will ever want to store more metadata w/ secondaries? if so maybe a json structure is better?

Reply:
i don't think so? as in it should always be an array of IDs, even if we switch away from POA